extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,256 @@
+# 3DTILES_metadata Extension
+
+## Contributors
+
+Cesium
+
+## Status
+
+Draft (version 0.0.0)
+
+## Overview
+
+This extension to 3D Tiles enables the declaration of metadata classes and the assignment of metadata to the tileset. Metadata may also be assigned to layers using [3DTILES_layers]().
+
+## Concepts
+
+### Metadata
+
+Metadata refers to information about the content of the tileset - it may be applicable to different components of 3D Tiles: tileset, layers and tiles.
+
+#### Classes
+
+Classes serve as the templates for the metadata objects - they provide a list of properties and the type information for those properties. For example, a tileset containing different layers of 3D data might create classes for each type:
+
+```javascript
+"classes": {
+  "DatasetClass": {
+    "properties": {
+      //...
+    }
+  },
+  "PhotogrammetryClass": {
+    "properties": {
+      //...
+    }
+  },
+  "BIMClass": {
+    "properties": {
+      //...
+    }
+  },
+  "PointCloudClass": {
+    "properties": {
+      //...
+    }
+  }
+}
+```
+
+This extensions uses the classes in compliance with the [Cesium 3D Metadata Specification]().
```

Comment:
Leaving a reminder here to eventually update these links to point to the right section of the metadata spec.

The spec will live in `3d-tiles/specification/Metadata/README.md` though it's probably best to wait until we finalize section headings before hyperlinking.